src/annotator/sidebar.js

```diff
@@ -283,6 +281,12 @@ export default class Sidebar {
         this._guestRPC.call('clearSelectionExceptIn', frameIdentifier);
       }
     );
+
+    if (this.bucketBar) {
+      this._guestRPC.on('anchorsChanged', () => {
+        this.bucketBar?.update();
+      });
+    }
```

Comment:
I was debating between:

```
    this._guestRPC.on('anchorsChanged', () => {
        this.bucketBar?.update();
    });
```

or

```
    if (this.bucketBar) {
      this._guestRPC.on('anchorsChanged', () => {
        /** @type{BucketBar} (this.bucketBar).update();
      });
    }
```

Wrapping the listener inside the `if` statement has the advantage to setup the listener if the `Sidebar` has a bucket bar.

Reply:
I prefer the first option you've presented here:

```
this._guestRPC.on('anchorsChanged', () => { this.bucketBar?.update(); });
```

As it's clear what's going on. If you leave the guarding conditional, can you add a comment?

From what I can see, the only reason a `Sidebar` would not instantiate a bucket bar is if it is in the clean theme. Is that correct?